### PR TITLE
Update sbs-gs-notificationbot.yml

### PR DESCRIPTION
**Pull request by mrackley**
It's probably obvious to most people, but there's a ton of helpful information in the readme file of the project for what to do next and how to extend the bot.  NO ONE wants to just deploy this bot and be done. They need to know what's next. Just add something at the bottom of the page to let them know they can do a lot more with it and there is help in the readme file.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d67edf9edc55352bc5ae0c8c6c96ae387ffef29f/msteams-platform%2Fsbs-gs-notificationbot.yml) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/edde069f-317c-40ff-8014-3cd5cee3923b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e63aa585106c6b61b0909a39398e439310b6d40f/msteams-platform%2Fsbs-gs-notificationbot.yml) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/483c3159-bb28-4f67-ae70-22e977b1dbdf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f62a7713797928c5b46cce95e6d1c4a31c099c19/msteams-platform%2Fsbs-gs-notificationbot.yml) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f4cd480-27be-4fd8-a428-b872f8a2f80d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-notificationbot.yml [hunk 1]
```diff
@@ -482,4 +482,5 @@ items:
      :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/trigger-output.png" alt-text="Trigger Output"border="true":::
 
 - content: |
-    You've completed the tutorial to build notification bot!    
+   You've completed the tutorial to build a notification bot. You can extend your notification bot to add the command response and workflow features.
+
```
Comment (MuyangAmigo) [inline msteams-platform/sbs-gs-notificationbot.yml:486]: Can we say something like "You can follow the README file or go to https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/command-bot-in-teams to further extend your app?"

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/732fc74dcbeebb6b9d247cb5e5eaafe1e0c03ffb/msteams-platform%2Fsbs-gs-notificationbot.yml) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d7a444f-f37b-4e76-ab97-dea522316271) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/886fe40557b48af6df9ea67b1c413a2608d51613/msteams-platform%2Fsbs-gs-notificationbot.yml) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05f93e54-620b-49ad-b2b8-17cbfac93fb7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1fc0c99e01b2c4cf27495b522a1cd6c905c68e1b/msteams-platform%2Fsbs-gs-notificationbot.yml) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb750be6-348d-448a-b398-c8fb712db6e2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/82db7d31b5e76dee2cd7868df5b236a6b28bf1d0/msteams-platform%2Fsbs-gs-notificationbot.yml) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/553802dc-c7cf-4069-baf2-f346486e5da8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-npaladugu — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-notificationbot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8f64afc932fa39487086f99edf9e61790ba824e7/msteams-platform%2Fsbs-gs-notificationbot.yml) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89c34a3c-14e1-443a-bdb3-5bd12fd50544) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**